src/clouds.cpp

```diff
@@ -58,13 +57,13 @@ Clouds::Clouds(scene::ISceneManager* mgr,
 	//m_material.MaterialType = video::EMT_TRANSPARENT_VERTEX_ALPHA;
 	m_material.MaterialType = video::EMT_TRANSPARENT_ALPHA_CHANNEL;
 
+	m_params.height        = cloudheight ? cloudheight : 120;
```

Comment:
That can be removed then, as a height of 100 was never used, that was always overriden by the default of 120, which is now added by your PR.